XML/TestCases/NestedVmTests.xml

```diff
@@ -424,6 +424,7 @@
     <Priority>2</Priority>
     <SetupConfig>
       <OSType>Windows</OSType>
+      <ARMImageName>MicrosoftWindowsServer windowsserver 2019-Datacenter latest,=~windowsserver</ARMImageName>
```

Comment:
Can you confirm this server version, Windows Server 2019 DataCenter is the baseline?

Reply:
This is not 'baseline'. Instead, this is the by default distro to be used for this test case, if user does not provide -ARMImageName from LISAv2 command.  And actually this test case accept any 'windowsserver'  sku, when testing with -ARMImageName parameter for gallery images (also accept -OsVHD when no -ARMImageName provided).   This test case will be ignored and skipped when user provide a Linux ARMImanageName, such as 'Canonical UbuntuServer 18.04-LTS latest', because it makes no sense.

Reply:
It's good to know. Based on the recent change of XML format, I suggest you put this information in XML comment or document. The customer won't know this combination and default version information unless we put down inside. Please add the brief OS requirement here or document.

Reply:
it's already included example in the README doc 'Note when testing with 'Azure' platform:'